docs/tutorials/scala/mxnet_scala_on_intellij.md

```diff
@@ -73,7 +73,6 @@ The configuration you should update is in the pom file's dependency for MXNet:
 <dependency>
   <groupId>org.apache.mxnet</groupId>
   <artifactId>mxnet-full_2.11-osx-x86_64-cpu</artifactId>
-  <version>1.2.0</version>
```

Comment:
In fact, if you remove this, maven is not gonna work. As I tried last time, you still have to specify a version. You can simply change this as a placeholder and let the user find the right version number from Maven central